Src/PerformanceCollector/Filtering.Shared/PortableHelpers.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.Filtering
+{
+    using System;
+    using System.Reflection;
+
+    internal static class PortableHelpers
+    {
+#if NET40
```

Comment:
Suggest to have file name include Net40 and instead of ifdef - have condition in include. Like this: https://github.com/Microsoft/ApplicationInsights-dotnet/blob/develop/src/Core/Managed/Core.csproj#L21